addons/udes_priorities/data/default_priority_groups.xml

```diff
@@ -0,0 +1,8 @@
+<odoo>
+  <data noupdate="1">
+    <record id="picking_group" model="udes_priorities.priority_group">
+      <field name="name">Picking</field>
+    </record>
+
+  </data>
+</odoo>
```

Comment:
Nit: No newline (same for `priorities_default_sequence.xml`).

Super nit: There is also an extra space on line 6